examples/wasip3-http-send-request/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+use axum::body::Body;
```

Comment:
Longer term it would be good to separate out the Axum sample from the "basic operations" samples I think?  Not a concern for the first drop, just gave me a bit of a start when I was looking for "how do I send a request" and Axum appeared in the answer!